### DISCUSSION TITLE
Testing that dialog/toast was opened?

### DISCUSSION THREAD

**Discussion by elijahcruz12** [tasteui/tasteui#444] [Q&A] [answered]
I want to write into my tests the ability to test that a dialog or toast (maybe even models if possible) where opened, as I'm trying to get 100% code coverage, but meanwhile I am, I want to be sure that when someone clicks my delete button, that the dialog does open with the correct question, since verifying this would be useful for client sites of mine, as some actually like to see that I can check this.

**Comment by devajmeireles** [comment, marked as answer]
If you are writing Dusk tests, I suggest you study our Dusk tests: 
https://github.com/tallstackui/tallstackui/tree/1.x/tests/Browser

If you are writing Unit tests, then you can only check if an event was dispatched with the appropriate data related to the action:
https://livewire.laravel.com/docs/testing#events
